app/helpers/current_user_helper.rb

```diff
@@ -5,7 +5,9 @@ def current_user_app?(name)
 
   def current_user_app(name)
     current_user_apps.values.find do |url|
-      url.include?("#{name}.prx.org") || url.include?("#{name}.staging.prx.tech")
+      url.include?("#{name}.prx.org") ||
+        url.include?("#{name}.staging.prx.tech") ||
+        url.include?("#{name}.prx.test")
```

Comment:
This "user app access detection" still feels a bit hacky to me.  I wonder how else we might do it (this is looking at the ID `/userinfo` endpoint, and what urls show up in there).

We could check the JWT scopes directly?  But then Feeder would have to ask for `augury:read-private` in its token, just so it could determine if you can go to Augury or not?

That also gets tricky when you have multiple accounts.  Maybe in one account, you can see Feeder but not Augury, and the exact opposite for other accounts.

I'll open a ticket.